modules/migration-using-must-gather.adoc

```diff
@@ -19,13 +27,33 @@ The `openshift-migration-must-gather-rhel8` image for {mtc-short} collects migra
 ----
 $ oc adm must-gather --image=registry.redhat.io/rhmtc/openshift-migration-must-gather-rhel8:v{mtc-version}
 ----
++
+The data is saved in a local `must-gather` directory.
```

Comment:
I think you can remove this comment. It's obvious that if you are in the directory and running the command there, the data is saved in that directory.

Reply:
Doesn't it mean that the command creates a new directory there - 'must-gather'?

Reply:
It creates a bunch of new directories with gz.zip files. You do not need to be that specific. Our end users have  cluster-admin privileges.

Reply:
So what is 'local `must-gather` directory'?

Reply:
Doesn't matter. Whatever the user wants.

Reply:
But there is no step where the user chooses a name for the directory that will be created. 
What I am trying to ask is this: Doesn't this comment about must-gather directory carry any useful information?
@apinnick

Reply:
No, I don't think it adds any useful information. It is fluff.